io.catenax.time_series_reference/1.0.0/TimeSeriesReference.ttl

```diff
@@ -0,0 +1,156 @@
+#######################################################################
+# Copyright (c) 2022 Allgemeine Deutsche Automobil-Club (ADAC) e.V
+# Copyright (c) 2022 Bayerische Motoren Werke Aktiengesellschaft
+# Copyright (c) 2022 Deutsches Zentrum für Luft- und Raumfahrt e. V. (DLR)
+# Copyright (c) 2022 Siemens AG
+# Copyright (c) 2022 ZF Friedrichshafen AG
+# Copyright (c) 2022 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This work is made available under the terms of the
+# Creative Commons Attribution 4.0 International (CC-BY-4.0) license,
+# which is available at
+# https://creativecommons.org/licenses/by/4.0/legalcode.
+#
+# SPDX-License-Identifier: CC-BY-4.0
+#######################################################################
+
+@prefix bamm: <urn:bamm:io.openmanufacturing:meta-model:2.0.0#> .
+@prefix bamm-c: <urn:bamm:io.openmanufacturing:characteristic:2.0.0#> .
+@prefix bamm-e: <urn:bamm:io.openmanufacturing:entity:2.0.0#> .
+@prefix unit: <urn:bamm:io.openmanufacturing:unit:2.0.0#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix : <urn:bamm:io.catenax.time_series_reference:1.0.0#> .
+
+:TimeSeriesReference a bamm:Aspect ;
+   bamm:preferredName "time series reference"@en ;
+   bamm:properties ( :decimalSeperator :sourceFile :fileExtension :delimiter :payload ) ;
+   bamm:operations ( ) ;
+   bamm:events ( ) ;
+   bamm:name "TimeSeriesReference" .
+
+:decimalSeperator a bamm:Property ;
+   bamm:preferredName "decimal seperator"@en ;
+   bamm:description "indicates whether a dot, comma or semicolon is used to separate the decimal place in the source file"@en ;
+   bamm:characteristic :DecimalSeperatorChoice ;
+   bamm:exampleValue "dot" ;
```

Comment:
consider using the actual character in the enum ie instead of "dot" -> ".". I assume the model will be used in an implementation and then this can directly be used